app/assets/stylesheets/widgets/single_column/messaging.scss

```diff
@@ -35,6 +35,74 @@
 		@include fluid-width-video-container;
 		display: inline-block;
 	}
+
+  .select {
+    position: relative;
+    margin-top: 10px; 
+    width: 100%;
+
+    .select-arrow {
+      position: relative;
+      left: 8px;
+      top: -3px;
+      display: inline-block;
+      background: transparent;
+      border-width: 0px 1px 1px 0px;
+      border-style: solid;
+      width: 8px;
+      height: 8px;
+      transform: rotateZ(45deg);
+      pointer-events: none;
+    }
+
+    .select-button {
+      width: 100%;
+      font-size: 16px;
+      line-height: 30px;
+      outline: none;
+      padding: $base-spacing/2 $base-spacing;
+      background-color: transparent;
+      box-shadow: none;
+      margin: 0;
+      height: auto;
+      border: 2px solid #000000;
+      border-radius: 3px;
+      background-position: center right;
+      background-repeat: no-repeat;
+
+      &.theme-text-color {
+
+      	#theme.dodge & {
```

Comment:
Can you add a comment about why this special set of rules is necessary for dodge